dataset_configs/spanish_pc/mcv12/config.yaml

```diff
@@ -126,17 +126,17 @@ processors:
 
   - _target_: sdp.processors.DuplicateFields
     input_manifest_file: "${workspace_dir}/manifests/${data_split}_fixed_text.json"
-    output_manifest_file: "${workspace_dir}/manifests/${data_split}_with_text_no_pc.json"
+    output_manifest_file: "${workspace_dir}/manifests/${data_split}_with_text_no_pc_a.json"
```

Comment:
I'd suggest to just remove all of these fields as they will be automatically assigned to tmp files. If anyone wants to look at intermediate configs, they can always add them back

Reply:
OK, I have removed all the specifications except for the first 2 processors (which may take a while), the "cleaned" manifest, and the "final" manifest.